src/RBACHandler.test.ts

```diff
@@ -175,16 +177,127 @@ describe('isAuthorized', () => {
     test('ERROR: Attempt to create a handler to support a new config version', async () => {
         expect(() => {
             // eslint-disable-next-line no-new
-            new RBACHandler({
-                version: 2.0,
-                groupRules: {},
-            });
+            new RBACHandler(
+                {
+                    version: 2.0,
+                    groupRules: {},
+                },
+                '4.0.1',
+            );
         }).toThrow(new Error('Configuration version does not match handler version'));
     });
 });
 
+describe('isAuthorized:Export', () => {
+    const customRBACRules: RBACConfig = {
+        version: 1.0,
+        groupRules: {
+            practitioner: {
+                operations: ['read'],
+                resources: [],
+            },
+        },
+    };
+
+    afterEach(() => {
+        customRBACRules.groupRules.practitioner = {
+            operations: ['read'],
+            resources: [],
+        };
+    });

```

Comment:
It's better to avoid sharing test data between tests. This implementation looks correct, but sharing test data makes it prone to breaking tests just by changing their order or by adding new tests.
 
I think it'd be better to have a function generate test data. i.e:
```js
const getTestPractitionerRBACRules = (operations, resources) => ({
        version: 1.0,
        groupRules: {
            practitioner: {
                operations,
                resources,
            },
        },
    });

...
const authZHandler: RBACHandler = new RBACHandler(
  getTestPractitionerRBACRules(['read'], ['Patient', 'MedicationRequest']),
  customRBACRules, '4.0.1'
);
``` 

you could have operations default to `['read']` but personally I find the tests more readable if the operations and resources are written on each test.

Reply:
fixed